dash/App/Accounts/Add/AddAddress/index.js

```diff
@@ -121,7 +127,7 @@ class AddAddress extends React.Component {
             <div className='addAccountItemOptionSetup' style={{ transform: `translateX(-${100 * this.state.index}%)` }}>
               <div className='addAccountItemOptionSetupFrames'>
                 <div className='addAccountItemOptionSetupFrame'>
-                  <div className='addAccountItemOptionTitle'>input address</div>
+                  <div className='addAccountItemOptionTitle'>input address or ENS domain</div>
```

Comment:
let's go with "ENS name" here instead of domain as that's how it's generally branded